ios/RainbowSplashScreen.swift

```diff
@@ -1,8 +1,8 @@
 import Foundation
 import React
 
-@objc(RainbowSplashScreenManager)
-class RainbowSplashScreenManager: NSObject {
+@objc(RainbowSplashScreen)
+class RainbowSplashScreen: NSObject {
 
   @objc
   func hideAnimated() {
```

Comment:
As mentioned in RN 0.79 upgrade PR, why do we delegate the `hideSplashScreenAnimated` to `appDelegate`? Maybe we should just move the implementation here directly?

Reply:
Let’s do this in another PR